src/foundry/client-esm/applications/apps/file-picker.d.mts

```diff
@@ -0,0 +1,396 @@
+import type { AnyObject, DeepPartial } from "../../../../utils/index.d.mts";
+import type ApplicationV2 from "../api/application.d.mts";
+import type HandlebarsApplicationMixin from "../api/handlebars-application.d.mts";
+import type { FormSelectOption } from "../forms/fields.d.mts";
+
+declare namespace FilePicker {
+  interface RenderContext extends ApplicationV2.RenderContext {
+    bucket: string | null;
+    buckets: FormSelectOption[] | null;
```

Comment:
Why is `undefined` not possible?

Reply:
output interfaces are always a little weird, but there's nothing to unify here so it's fine.